pkg/config/config_template.yaml

```diff
@@ -1011,6 +1011,12 @@ api_key:
 #
 # dogstatsd_socket: ""
 
+## @param dogstatsd_telemetry_endpoint - string - optional - default: ""
```

Comment:
I think `dogstatsd_telemetry_listener` would be a more accurate name for this option